### PR TITLE
Team2 - move sign in/up links to footer

### PR DESCRIPTION
**Pull request by ghost**
Team 3 please review

### CODE REVIEW

File: app/views/layouts/application.html.erb [hunk 1]
```diff
@@ -1,95 +1,94 @@
...
-</head>
-<body>
+    <head>
+	<title>VancouverTechHub</title>
+	<%= stylesheet_link_tag    'application', media: 'all', 'data-turbolinks-track' => true %>
+	<%= javascript_include_tag 'application', 'data-turbolinks-track' => true %>
+	<%= csrf_meta_tags %>
+	<script src="//maps.google.com/maps/api/js?v=3.23&sensor=false&client=&key=&libraries=geometry&language=&hl=&region="></script>
+	<script src="//cdn.rawgit.com/mahnunchik/markerclustererplus/master/dist/markerclusterer.min.js"></script>
+	<script src='//cdn.rawgit.com/printercu/google-maps-utility-library-v3-read-only/master/infobox/src/infobox_packed.js' type='text/javascript'></script> <!-- only if you need custom infoboxes -->
```
Comment (tkbeili) [inline app/views/layouts/application.html.erb]: Two space indentations. Please fix.

File: app/views/layouts/application.html.erb [hunk 2]
```diff
@@ -1,95 +1,94 @@
...
+	<nav class="navbar navbar-inverse navbar-fixed-top">
+	    <div class="container-fluid">
+		<a href="<%= root_path%>" class="navbar-brand" id="logo"> <%= image_tag("logo_white.png")%></a>
+		<div class="navbar-header">
+		    <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#responsive-menu">
+			<span class="sr-only">Toggle navigation</span>
+			<span class="icon-bar"></span>
+			<span class="icon-bar"></span>
+			<span class="icon-bar"></span>
+		    </button>
```
Comment (tkbeili) [inline app/views/layouts/application.html.erb]: Two-space indentations please fix.

File: app/views/layouts/application.html.erb [hunk 3]
```diff
@@ -1,95 +1,94 @@
...
+		<a href="https://github.com/CodeCoreYVR/vancouvertechhub" alt="GitHub" target="_blank" ><i class="fa fa-github" aria-hidden="true"></i></a>
+		<a href="https://twitter.com/CodeCoreYVR" alt="Twitter" target="_blank" ><i class="fa fa-twitter" aria-hidden="true"></i></a>
+		<a href="https://www.facebook.com/CodeCoreBootcamp" alt="Facebook" target="_blank"><i class="fa fa-facebook-official" aria-hidden="true"></i></a>
+		<a href="https://plus.google.com/+CodecoreCaBootcamp/about" alt="Google+" target="_blank"><i class="fa fa-google-plus" aria-hidden="true"></i></a>
+	    </div>
+	    <div class="navbar-right footer-text">
+		Built and maintained by students from <a href="http://codecore.ca" alt="Codecore" target="_blank">CodeCore</a>
+	    </div>
+	</div>
+    </body>
```
Comment (tkbeili) [inline app/views/layouts/application.html.erb]: Please fix indentations.

**Comment by tkbeili** [comment]
👍 Looks good. Get another +1 from Team 3

**Comment by whilestevego** [comment]
This PR only has one thumbs up from Team 3. It needs another before passing it on to me. @yatmchang, did you review this PR after Tam mentioned Team 3?

**Comment by williamtstanley** [comment]
There is another thumbs up now.

**Comment by yatmchang** [comment]
Ya I did @whilestevego

**Comment by whilestevego** [comment]
Looking good! This is ready to merge.

**Pull request merged**